pep/app/pods/components/page/sidebar/component.ts

```diff
@@ -29,11 +70,86 @@ export default class PageSidebar extends Component<PageSidebarArgs> {
         return this.isLeft ? this.sidebar.rightSidebarIsOpen : this.sidebar.leftSidebarIsOpen;
     }
 
+    get sidebarStyles() {
+        return this.resizedWidth && this.isOpen
+            ? htmlSafe(`width: ${this.resizedWidth + SIDEBAR_HANDLE_WIDTH}px;`)
+            : null;
```

Comment:
I don't actual know the answer without experimenting .. but can you just returned undefined instead of null?

Reply:
probably should work, but we typically always pass `null` to `style=` attributes in ember when we dont want any styles rendered, any reason you prefer `undefined` here instead?

Reply:
It would allow you make this code more concise, transforming it to

```
get sidebarStyles() {
    return this.resizedWidth && this.isOpen && htmlSafe(`width: ${this.resizedWidth + SIDEBAR_HANDLE_WIDTH}px;`)
}
```

Reply:
in that example, if `resizedWidth` is zero, it would return zero, or if `isOpen` is false, it would return false, so I'd feel less confident about returning values like that to the style attribute.